app/models/Signup.js

```diff
@@ -129,8 +131,10 @@ signupSchema.statics.lookupCurrent = function (user, campaign) {
       .then(signupDoc => resolve(signupDoc))
       .catch((err) => {
         app.locals.stathatError(statName, err);
```

Comment:
Ok. I was looking at this tab in stathat: https://www.stathat.com/v/stats/Zxka and thought the message would help clear up what this error means "bad gateway". I maybe just have to get more familiar with how all this errors are being reported.